lib/elixir/lib/module/types/descr.ex

```diff
@@ -254,25 +234,45 @@ defmodule Module.Types.Descr do
     |> IO.iodata_to_binary()
   end
 
-  ## Iterator helpers
+  ## Map helpers
```

Comment:
I would move those helpers to the bottom of the file. :)

Reply:
[86e478b](https://github.com/elixir-lang/elixir/pull/13524/commits/86e478bf6c1c785b844f5d3e44d81372877a38e8) ✅